### PR TITLE
Update pl.json

### PR DESCRIPTION
**Pull request by grzegorz914**
## :recycle: Current situation

*Describe the current situation. Explain current problems, if there are any. Be as descriptive as possible (e.g., including examples or code snippets).*

## :bulb: Proposed solution

*Describe the proposed solution and changes. How does it affect the project? How does it affect the internal structure (e.g., refactorings)?*

## :gear: Release Notes

*Provide a summary of the changes or features from a user's point of view. If there are breaking changes, provide migration guides using code examples of the affected features.*

## :heavy_plus_sign: Additional Information
*If applicable, provide additional context in this section.*

### Testing

*Which tests were added? Which existing tests were adapted/changed? Which situations are covered, and what edge cases are missing?*

### Reviewer Nudging

*Where should the reviewer start? what is a good entry point?*

### CODE REVIEW

File: ui/src/i18n/pl.json [hunk 1]
```diff
@@ -50,28 +50,28 @@
   "accessories.message_settings_link": "Możesz uzyskać dostęp do ustawień Homebridge w dowolnym momencie z menu głównego lub klikając cog",
   "accessories.title_accessories": "Akcesoria",
   "accessories.title_accessory_control_disabled": "Sterowanie akcesoriami jest wyłączone",
-  "backup.button_download_backup_archive": "Utwórz backup",
-  "backup.button_restore_backup": "Przywróć backup",
-  "backup.label_archive_name": "Nazwa backupu",
-  "backup.label_backup_size": "Rozmiar backupu",
+  "backup.button_download_backup_archive": "Utwórz kopię zapasową",
```
Comment (mkz212) [inline ui/src/i18n/pl.json]: "Backup" is definitely more popular than "kopia zapasowa".

File: ui/src/i18n/pl.json [hunk 2]
```diff
@@ -50,28 +50,28 @@
...
-  "backup.message_restore_warning": "Przywrócenie backupu jest nieodwracalne i zastąpi całą obecną konfigurację Homebridge.",
-  "backup.message_scheduled_backup_none_yet": "Nie utworzono jeszcze planowych backupów. Następny automatyczny backup zostanie wykonany o godzinie {{ backupTime }}.",
-  "backup.message_scheduled_backup_time": "Pełny backup konfiguracji Homebridge jest tworzony automatycznie każdego dnia o godzinie {{ backupTime }} i trzymany przez {{ dayCount }} dni.",
-  "backup.title_backup": "Backup - utwórz",
-  "backup.title_restore": "Backup - przywróć",
+  "backup.message_restore_warning": "Przywrócenie kopii zapasowej jest nieodwracalne i zastąpi całą obecną konfigurację Homebridge.",
+  "backup.message_scheduled_backup_none_yet": "Nie utworzono jeszcze planowych kopii zapasowych. Następny automatyczny backup zostanie wykonany o godzinie {{ backupTime }}.",
+  "backup.message_scheduled_backup_time": "Pełna kopia zapasowa konfiguracji Homebridge jest tworzona automatycznie każdego dnia o godzinie {{ backupTime }} i trzymany przez {{ dayCount }} dni.",
+  "backup.title_backup": "Kopia zapasowa",
```
Comment (mkz212) [inline ui/src/i18n/pl.json]: "backup.title_backup" and "backup.title_backup_restore" - you use same title, that is not correct, because one is to CREATE backup and second is to RESTORE backup.

File: ui/src/i18n/pl.json (hunk 1 above)
Comment (grzegorz914) [reply ui/src/i18n/pl.json]: I think „Backup” it’s not a Polish name. I think is English, right?

File: ui/src/i18n/pl.json [hunk 3]
```diff
@@ -225,10 +225,10 @@
   "plugins.manage.message_uninstall_remove_confirmation": "Na pewno odinstalować {{ pluginName }}?",
   "plugins.manage.message_uninstall_remove_plugin_config": "Usunąć również konfigurację tej wtyczki?",
   "plugins.manage.modal_unverified_message": "Niezweryfikowane wtyczki Homebridge wskazują, że nie zostały poddane procesowi przeglądu przez zespół projektowy Homebridge. Brak weryfikacji niekoniecznie oznacza nieskuteczność lub obawy dotyczące bezpieczeństwa. Autorzy mogli zdecydować się nie szukać weryfikacji z różnych powodów.",
-  "plugins.manage.modal_unverified_subtitle": "{{ pluginName }} jest niezweryfikowane.",
+  "plugins.manage.modal_unverified_subtitle": "{{ pluginName }} jest niezweryfikowany.",
```
Comment (mkz212) [inline ui/src/i18n/pl.json:228]: "niezweryfikowany" or "niezweryfikowana"

File: ui/src/i18n/pl.json [hunk 4]
```diff
@@ -225,10 +225,10 @@
...
   "plugins.manage.modal_unverified_title": "Niezweryfikowane wtyczki",
   "plugins.manage.modal_verified_message": "Wtyczki Homebridge oznaczone jako zweryfikowane zostały sprawdzone przez zespół Homebridge aby upewnić się, że spełniają różne wymagania, które zachęcają do najlepszych praktyk i bezproblemowego doświadczenia użytkownika.",
-  "plugins.manage.modal_verified_subtitle": "{{ pluginName }} jest zweryfikowane.",
+  "plugins.manage.modal_verified_subtitle": "{{ pluginName }} jest zweryfikowany.",
```
Comment (mkz212) [inline ui/src/i18n/pl.json]: "zweryfikowany" or "zweryfikowana"

File: ui/src/i18n/pl.json [hunk 5]
```diff
@@ -307,7 +307,7 @@
   "settings.reset.help": "Jeśli masz problemy z podłączeniem Homebridge lub zewnętrznych akcesoriów takich jak kamery lub TV z HomeKit, być może będziesz musiał zresetować Homebridge.",
   "settings.service.debug_tooltip": "Niektóre wtyczki wymagają ustawienia zmiennej środowiskowej DEBUG, aby umożliwić rejestrowanie poziomu debugowania.",
   "settings.service.node_tooltip": "Opcje uruchomieniowe Node.js dla procesu Homebridge. To powinno być puste, chyba że wiesz, co robisz.",
-  "settings.startup.debug": "Tryb Homebridge Debug",
+  "settings.startup.debug": "Debugowanie Homebridge",
```
Comment (mkz212) [inline ui/src/i18n/pl.json]: This is a mode designation, not an action.

File: ui/src/i18n/pl.json [hunk 6]
```diff
@@ -405,9 +405,9 @@
   "support.developers.item_swagger_sub": "Homebridge REST API umożliwia interakcję z serwerem Homebridge za pomocą żądań HTTP.",
   "support.developers.template": "Repozytorium szablonu wtyczki",
   "support.developers.template_sub": "Szablon wtyczki Homebridge, który może być używana jako podstawa, aby pomóc Ci rozpocząć tworzenie własnej wtyczki.",
-  "support.developers.title": "Deweloper",
-  "support.developers.verified": "Verified by Homebridge",
-  "support.developers.verified_sub": "The Verified By Homebridge pozwala twórcom wtyczek na sprawdzenie i zatwierdzenie ich przez zespół projektowy Homebridge.",
+  "support.developers.title": "Twórca",
```
Comment (mkz212) [inline ui/src/i18n/pl.json]: Developer is not author. Author is creator of the plugin, developer It may be someone who develops the plugin but does not necessarily create it.

File: ui/src/i18n/pl.json [hunk 7]
```diff
@@ -263,7 +263,7 @@
   "reset.message_need_to_remove_homebridge_accessory_from_home_app": "Będziesz musiał ręcznie usunąć istniejące akcesoria Homebridge z aplikacji domowej.",
   "reset.message_remove_cached_accessories": "Ta czynność całkowicie usunie wszystkie akcesoria z pamięci podręcznej.",
   "reset.message_remove_cached_accessories_single": "Ta czynność usunie jedno z akcesoriów z pamięci podręcznej.",
-  "reset.message_remove_cached_accessories_single_empty": "Nie masz żadnych buforowanych akcesoriów do usunięcia.",
+  "reset.message_remove_cached_accessories_single_empty": "Nie posiadasz akcesoriów w pamięci podręcznej do usunięcia.",
```
Comment (mkz212) [inline ui/src/i18n/pl.json]: "Nie posiadasz w pamięci podręcznej żadnych akcesoriów do usunięcia."

File: ui/src/i18n/pl.json [hunk 8]
```diff
@@ -385,8 +385,8 @@
   "status.widget.systeminfo.label_yes": "Tak",
   "status.widget.systeminfo.modal_glibc_message": "Ten komunikat wskazuje, że twój system operacyjny nie obsługuje nowszych wersji Node.js. Aby rozwiązać ten problem i móc zainstalować zaktualizowane wersje Node.js w przyszłości, trzeba zaktualizować system do nowszej wersji.",
   "status.widget.systeminfo.modal_glibc_title": "Aktualizacja OS",
-  "status.widget.systeminfo.modal_node_unsupp_message": "Uruchamiasz wersję Node.js, która nie jest oficjalnie obsługiwana przez Homebridge. Powinieneś rozważyć zmianę na wspieraną wersję.",
-  "status.widget.systeminfo.modal_node_unsupp_title": "Nieobsługiwana wersja Node.js",
+  "status.widget.systeminfo.modal_node_unsupp_message": "Używasz wersji Node.js która nie jest oficjalnie wspierana przez Homebridge. Powinieneś użyć wersji oficjalnie wspieranej.",
+  "status.widget.systeminfo.modal_node_unsupp_title": "Niewspietana wersja Node.js",
```
Comment (mkz212) [inline ui/src/i18n/pl.json]: "Niewspierana"

File: ui/src/i18n/pl.json [hunk 9]
```diff
@@ -405,9 +405,9 @@
   "support.developers.item_swagger_sub": "Homebridge REST API umożliwia interakcję z serwerem Homebridge za pomocą żądań HTTP.",
   "support.developers.template": "Repozytorium szablonu wtyczki",
   "support.developers.template_sub": "Szablon wtyczki Homebridge, który może być używana jako podstawa, aby pomóc Ci rozpocząć tworzenie własnej wtyczki.",
-  "support.developers.title": "Deweloper",
-  "support.developers.verified": "Verified by Homebridge",
-  "support.developers.verified_sub": "The Verified By Homebridge pozwala twórcom wtyczek na sprawdzenie i zatwierdzenie ich przez zespół projektowy Homebridge.",
+  "support.developers.title": "Twórca",
+  "support.developers.verified": "Zweryfikowano przez Homebridge",
```
Comment (mkz212) [inline ui/src/i18n/pl.json]: "Verified by" It's a kind of brand, it shouldn't be translated

File: ui/src/i18n/pl.json (hunk 3 above)
Comment (grzegorz914) [reply ui/src/i18n/pl.json:228]: Camera UI jest niezweryfikowana, 
WebOS Control jest niezweryfikowany, 
Wtyczki są niezweryfikowane

I think first two are correct

File: ui/src/i18n/pl.json (hunk 4 above)
Comment (grzegorz914) [reply ui/src/i18n/pl.json]: same as before

File: ui/src/i18n/pl.json (hunk 1 above)
Comment (mkz212) [reply ui/src/i18n/pl.json]: You say in Poland "hot-dog" or "gorący pies"? :)

File: ui/src/i18n/pl.json (hunk 5 above)
Comment (grzegorz914) [reply ui/src/i18n/pl.json]: It’s not a switch? which enable debug mode for homebridge.

File: ui/src/i18n/pl.json (hunk 3 above)
Comment (mkz212) [reply ui/src/i18n/pl.json:228]: Zrzut ekranu 2024-07-25 o 09 22 55

This is not displayed next to the plugin name. So for me "niezweryfikowana" or "nie zweryfikowano" (with space, cause it is verb).

File: ui/src/i18n/pl.json (hunk 5 above)
Comment (mkz212) [reply ui/src/i18n/pl.json]: Yes, it is switch to enable mode, not to perform one time action.

File: ui/src/i18n/pl.json [hunk 10]
```diff
@@ -51,27 +51,27 @@
...
   "backup.message_restore_help_two": "Po przywróceniu backupu trzeba ponownie zalogować się do Homebridge UI z danymi użytkownika zawartymi w tej kopii.",
-  "backup.message_restore_max_size": "Możesz przywrócić archiwum kopii zapasowej do rozmiaru: {{ maxBackupSizeText }}.",
+  "backup.message_restore_max_size": "Możesz przywrócić backup do rozmiaru: {{ maxBackupSizeText }}.",
   "backup.message_restore_warning": "Przywrócenie backupu jest nieodwracalne i zastąpi całą obecną konfigurację Homebridge.",
   "backup.message_scheduled_backup_none_yet": "Nie utworzono jeszcze planowych backupów. Następny automatyczny backup zostanie wykonany o godzinie {{ backupTime }}.",
-  "backup.message_scheduled_backup_time": "Pełny backup konfiguracji Homebridge jest tworzony automatycznie każdego dnia o godzinie {{ backupTime }} i trzymany przez {{ dayCount }} dni.",
-  "backup.title_backup": "Backup - utwórz",
-  "backup.title_restore": "Backup - przywróć",
+  "backup.message_scheduled_backup_time": "Pełny backup konfiguracji Homebridge jest tworzona automatycznie każdego dnia o godzinie {{ backupTime }} i trzymany przez {{ dayCount }} dni.",
+  "backup.title_backup": "Backup",
```
Comment (mkz212) [inline ui/src/i18n/pl.json]: "Utwórz backup"

Comment (mkz212) [reply ui/src/i18n/pl.json]: Although I personally prefer "Backup - utwórz" and "Backup - przywróć"

Comment (mkz212) [reply ui/src/i18n/pl.json]: IMG_6479

**Comment by mkz212** [comment]
@grzegorz914 Thank you for taking into account my suggestions! 👍

**Comment by mkz212** [comment]
@grzegorz914 

line 67 - 'accessorimai' could you correct to 'akcesoriami' ?

**Comment by mkz212** [comment]
@grzegorz914 'akcesoriami'

**Comment by mkz212** [comment]
@grzegorz914 Thank you. 👍
